src/steps/finish/upload.tsx

```diff
@@ -205,14 +214,6 @@ const UploadForm: React.FC<UploadFormProps> = ({ handleUpload }) => {
     }
   }
 
-  // If the user has not yet changed the value of the field and the last used
-  // presenter name is used in local storage, use that.
-  useEffect(() => {
-    if (presenterValue !== presenter) {
-      dispatch({ type: "UPDATE_PRESENTER", value: presenterValue });
-    }
-  });
-
```

Comment:
Wait, why was this deleted?

Reply:
Because I was overly eager while cleaning up. 😅 It's back now, but I took the liberty of touching it up a bit.